### ISSUE TITLE
Facebook/Twitter post is being blocked by security

### ISSUE THREAD

**Issue by silasmeredith** [DevProgress/more-tacos#82]
![2016-09-12_12-03-33](https://cloud.githubusercontent.com/assets/22085972/18443021/1c1261ec-78e1-11e6-8cb2-df5c930c6e1b.jpg)

**Comment by kielni** [comment]
did you click the 'let us know' link? I have no idea what it it doesn't like.

**Comment by kielni** [comment]
from https://developers.facebook.com/tools/debug/sharing/?q=http%3A%2F%2Ftacotruck.party

![screen shot 2016-09-12 at 10 03 49 am](https://cloud.githubusercontent.com/assets/1649528/18444966/3c226308-78d0-11e6-8adc-e0c0317314b2.png)

**Comment by nathanaelphilip** [comment]
/lat/{lat}/lng/{lng} ok ?

**Comment by kielni** [comment]
I merged it but it looks like somewhere is still adding the hash
Can we use single letters for the param names? The long lat/lng value taking up most of the tweet character count.  I know they shorten it but it looks like the popup is counting the full length.

<img width="517" alt="screen shot 2016-09-12 at 5 54 06 pm" src="https://cloud.githubusercontent.com/assets/1649528/18457973/ef7aa29e-7911-11e6-8bda-ed10037c94c4.png">

**Comment by Eyas** [comment]
@kielni I looked into this but it seems it only happens locally because it fails to parse the URL as a URL. Try adding a `.com` after `localhost` in the twitter box and you will see the character count go back to normal :)

**Comment by silasmeredith** [comment]
Just for housekeeping, I updated the title of this issue to refer to both Facebook and Twitter. The Facebook issue seems to be solved (works for me, anyway), but we still need to figure out Twitter.

Currently, the lat/long link included in the Twitter share is the default lat/long, not the specific lat/long.Is that the issue that y'all are discussing above? If not, let me know so I can open a separate ticket.

**Comment by kielni** [comment]
Twitter is working for me; https://github.com/DevProgress/more-tacos/issues/45 covers it so this one can be closed.
